spec.md

```diff
@@ -167,6 +167,57 @@ merged are hosted in other folders.
 As some Compose file elements can both be expressed as single strings or complex objects, merges MUST apply to
 the expanded form.
 
+### Profiles
+
+Profiles allow to adjust the Compose application model for various usages and environments. A Compose
+implementation SHOULD allow the user to define a set of active profiles. The exact mechanism is implementation
+specific and MAY include command line flags, environment variables, etc.
+
+The Services top-level element supports a `profiles` attribute to define a list of named profiles. Services without
```

Comment:
imho this should not be restricted to `services`, but applied to all top-level elements

Reply:
For _this_ PR I reduced it to services to keep the scope as small as possible while it also already covers a majority of the use cases with the intention to expand with future PRs :+1:

Reply:
I think that's a great idea to limit the scope 🙌 To avoid ambiguity, should we clarify what happens to other objects in this case? I guess this can be as simple as specifying that all networks/volumes/etc are treated as always enabled for now.

Reply:
`All other top-level elements are not affected by `profiles` and are always active.`

This is now addressed 👍